src/mito2/src/memtable/merge_tree/tree.rs

```diff
@@ -316,6 +319,15 @@ impl MergeTree {
 struct TreeIter {
     partitions: VecDeque<PartitionRef>,
     current_reader: Option<PartitionReader>,
+    fetch_partition_elapsed: Duration,
```

Comment:
Could we also collect how many rows and batches we returned? We might also log them in debug logs.

Reply:
> Could we also collect how many rows and batches we returned? We might also log them in debug logs.

fixed https://github.com/GreptimeTeam/greptimedb/pull/3397/commits/8a1736bc7931ec9ffaf44962aff46713c5f0bc91#diff-c49c8c96fe96144915d2d855220956e3284752035404a1a9b230934e2ec213deR389-R390